RobotCrawler/src/main/java/parser/kobobooks/KoboBooksParser.java

```diff
@@ -0,0 +1,32 @@
+package parser.kobobooks;
+
+import model.BookData;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class KoboBooksParser {
+
+    private static final String freeBooksUrl = "/p/free-ebooks";
+    KoboBooksPageParser booksPageParser;
+    KoboBooksCategoryPageParser booksCategoryPageParser;
+    KoboBooksCategoryParser booksCategoryParser;
```

Comment:
Maybe KoboBooksPageParser can be renamed to KoboBooksCategoriesUrlParser.
KoboBooksCategoryPageParser can be renamed to KoboBooksUrlParser.
KoboBooksCategoryParser can be renamed to KoboBooksParser.
But I have not idea how to change name of KoboBooksParser.

This is only suggestion :)